### PR TITLE
<Slider> - add aria-label

### PR DESCRIPTION
**Pull request by mosheKerbel**
@argshook @shlomitc 
will open an issue in wix-ui-santa, the only consumers, after merged.

### CODE REVIEW

File: packages/wix-ui-core/src/components/slider/Slider.meta.ts [hunk 1]
```diff
@@ -5,5 +5,7 @@ const sliderMetadata = Registry.getComponentMetadata(Slider);
 sliderMetadata.nonA11yCompliant = true;
```
Comment (argshook) [inline packages/wix-ui-core/src/components/slider/Slider.meta.ts]:
```suggestion
```
@mosheKerbel i think a11y test should work now?

Comment (mosheKerbel) [reply packages/wix-ui-core/src/components/slider/Slider.meta.ts]: yeah that was the whole point..a glitch
will fix

**Comment by mosheKerbel** [comment]
ignoring the wix-ui-framework failure not related
also does the tooltop test that fails

**Comment by mosheKerbel** [comment]
apprvoed by Arijus.

**Pull request merged**